src/utils/index.js

```diff
@@ -91,22 +91,20 @@ export const blockToDate = (goalBlock, height) => {
   return new Date(diff * 180000 + Date.now());
 };
 
-export const wrapTry = async (promise, store) => {
+// eslint-disable-next-line consistent-return
+export const wrapTry = async (promise) => {
   try {
     return promise.then((res) => {
       if (!res) {
-        if (store) { store.commit('setBackendStatus', false); }
-        return null;
+        throw new Error('No response object');
       }
-      if (store) { store.commit('setBackendStatus', true); }
-      if (!res.ok) throw new Error(`Request failed with ${res.status}`);
+      if (res.status !== 200) {
+        throw new Error(`Error code ${res.status}`);
+      }
+      if (!res.ok) throw new Error(4000);
       return res.json();
-    }).catch((error) => {
-      console.error(error);
-      return null;
     });
   } catch (err) {
```

Comment:
You don't need an error handler if you are just passing an error through.

Reply:
this one doesn't looks like resolved

Reply:
resolved thanks

Reply:
@davidyuk so now the error is thrown automatically by the fetch promise and handled next at the plugin

Reply:
You don't need an error handler if you are just passing an error through.

Reply:
@davidyuk I removed the whole try catch and let the error throw without that definition